Documentation/Dockerfile

```diff
@@ -2,26 +2,16 @@ FROM docker.io/library/python:3.7.5-alpine3.10
 
 LABEL maintainer="maintainer@cilium.io"
 
-ENV DOCS_DIR=/srv/Documentation
-ENV API_DIR=/srv/api
-
-ADD ./requirements.txt $DOCS_DIR/requirements.txt
-
-ENV PACKAGES="\
+RUN apk add --no-cache --virtual --update \
+    aspell-en \
     bash \
     ca-certificates \
-    make \
+    enchant \
     git \
-    python \
     py-pip \
+    python \
     sphinx-python \
-    enchant \
-    aspell-en \
-"
-RUN apk add --no-cache --virtual --update $PACKAGES && \
-    pip install --upgrade pip && \
-    pip install -r $DOCS_DIR/requirements.txt
+    && true
 
-WORKDIR $DOCS_DIR
-ADD _api $API_DIR
-ADD . $DOCS_DIR
+ADD ./requirements.txt /tmp/requirements.txt
+RUN pip install -r /tmp/requirements.txt
```

Comment:
GitHub says end-of-line is missing?

Reply:
Seems like a standard vscode behaviour.. let me see if there is a setting for it.